core/templates/dev/head/domain/utilities/UrlInterpolationService.ts

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+require('services/contextual/UrlService.ts');
```

Comment:
Move this after the other requires please.

Reply:
By alphabetical order, I think it should be in the middle